### PR TITLE
implement new query service method

### PR DESCRIPTION
**Pull request by filipdujmusic**

### CODE REVIEW

File: contracts/services/QueryService.sol [hunk 1]
```diff
@@ -111,6 +111,12 @@ interface IQueryService is IVersioned {
         address[] memory factories,
         INameRegistry nameRegistry
     ) external view returns (Structs.AssetCommonStateWithName[] memory);
+
+    function getAssetStatesForIssuer(
```
Comment (githinho) [inline contracts/services/QueryService.sol:115]:
```suggestion
    function getAssetsWtihStateForIssuer(
```

`getAssetStatesForIssuer` sounds like you are trying to fetch to different states for assets.

File: contracts/services/QueryService.sol [hunk 2]
```diff
@@ -440,14 +446,45 @@ contract QueryService is IQueryService {
         return response; 
     }
 
+    function getAssetStatesForIssuer(
```
Comment (githinho) [inline contracts/services/QueryService.sol:449]:
```suggestion
    function getAssetsStateWithStateForIssuer(
```

File: test/services/query-service-test.ts [hunk 3]
```diff
@@ -68,6 +71,133 @@ describe("Query service tests", function () {
...
+            false, false ,false,
+            assetBasicName,
+            assetBasicSymbol,
+            assetBasicInfoHash,
+            testData.assetFactory,
+            testData.nameRegistry,
+            testData.apxRegistry
+        );
+
+
```
Comment (githinho) [inline test/services/query-service-test.ts:107]: Double spacing/spending.

File: test/services/query-service-test.ts [hunk 4]
```diff
@@ -68,6 +71,133 @@ describe("Query service tests", function () {
...
+            testData.issuer.address,
+            [
+                testData.assetFactory.address,
+                testData.assetTransferableFactory.address,
+                testData.assetSimpleFactory.address
+            ], [ ]
+        );
+        expect(emptyIssuerResponse.length).to.be.equal(0);
+
+        const assetBasicName = "Test Asset";
```
Comment (githinho) [inline test/services/query-service-test.ts:88]:
```suggestion
        const assetBasicName = "Alen Basic";
```

**Review by domagojlatecki — APPROVED** [review]
Nothing to add, but please address comments by @githinho

**Pull request merged**